src/components/photoGallery/PhotoGallery.tsx

```diff
@@ -1,8 +1,10 @@
 import React, { useState, useEffect } from "react";
 import "./PhotoGallery.scss";
+import { LargeImage } from "./LargeImage";
 
 export const PhotoGallery: React.FunctionComponent = () => {
   const [imageUrls, setImageUrls] = useState<string[]>();
+  const [largeImageUrl, setLargeImageUrl] = useState<string>("https://apod.nasa.gov/apod/image/0201/earthrise_apollo8.jpg");
```

Comment:
Don't need to specify type when it can be inferred from the initial value
```suggestion
  const [largeImageUrl, setLargeImageUrl] = useState("https://apod.nasa.gov/apod/image/0201/earthrise_apollo8.jpg");
```